src/oop/app/src/Scrapper.php

```diff
@@ -1,9 +1,53 @@
 <?php
+
 /**
  * Create Class - Scrapper with method getMovie().
  * getMovie() - should return Movie Class object.
  *
  * Note: Use next namespace for Scrapper Class - "namespace src\oop\app\src;"
- * Note: Dont forget to create variables for TransportInterface and ParserInterface objects.
+ * Note: Dont forget to create variables for
+ * TransportInterface and ParserInterface objects.
  * Note: Also you can add your methods if needed.
  */
+
+namespace src\oop\app\src;
+
+use src\oop\app\src\Models\Movie;
+use src\oop\app\src\Parsers\ParserInterface;
+use src\oop\app\src\Transporters\TransportInterface;
+
+/**
+ * Initializes own private vars $Transporter and $Parser l by supplied objects
+ * and using them in method GetMovie to get and parce content to set Movie obgect
+ * Scrapper
+ */
+class Scrapper
+{
+    private mixed $transporter;
+    private mixed $parser;
+    private mixed $movie;
+
+    /**
+     * Sets transporter and parser objects for future getMogie routine
+     * @param  TransportInterface  $transporter object
+     * @param  ParserInterface  $parser object
+     */
+    public function __construct(TransportInterface $transporter, ParserInterface $parser)
+    {
+        $this->transporter = $transporter;
+        $this->parser = $parser;
+        $this->movie = new Movie();
```

Comment:
please pass Movie as dependency (do not use new Movie())

Reply:
Done